### ISSUE TITLE
[DOC] Use a different documentation theme

### ISSUE THREAD

**Issue by teald** [GeminiDRSoftware/astrodata#52]
## Description
There are issues with implementing better accessibility features, such as 
+ dark mode (see #29)
+ There doesn't seem to be much documentation of how/if ReadTheDocs considers accessibility on its platform
  + I need to do more research here, but a relatively deep search hasn't yielded anything yet.

While this may be as simple as implementing a style sheet and calling it a day, I worry there are other accessibility issues that the RTD theme does not have the resources to deal with in the near future. If it does become a matter of using different CSS then it may be worth opening a PR there (assuming it's compatible).

## Solution
[If applicable, your solution to the issue]

### Urgency
- [x] This problem should be fixed quickly (urgent)
- [ ] This problem is minor/does not require immediate fixing.
